tt_metal/include/compute_kernel_api/eltwise_unary/remainder.h

```diff
@@ -0,0 +1,46 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+
+#include "compute_kernel_api/common_globals.h"
+#ifdef TRISC_MATH
+#include "llk_math_eltwise_unary_sfpu_remainder.h"
+#define MAIN math_main()
+#define MATH(x) x
+#else
+#define MATH(x)
+#endif
+
+
+
+namespace ckernel {
+
+/**
+ * Performs element-wise modulus computation on input x by y , where x is each element of a tile
+ * in DST register at index tile_index. The input must be of int data type only. The value is provided as const param0 The DST register buffer must be in
```

Comment:
Should we be asserting for int dtype for the input to remainder in our host op code since this says it needs to be int dtype?